styleguide/source/_patterns/03-organisms/search-result-list.md

```diff
@@ -0,0 +1,32 @@
+---
+el: ".ama__search-result-list"
+title: "Search Result List"
+---
+
+A collection of up to ten search results and one search result as best bet displayed in a column (meant for use on the Search Results page).
```

Comment:
Based on prior conversations, I believe best bets restult is never shown within search results. This description should be adjusted to reflect that.